tests/Export/ProductServiceTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Tests\Export;
+
+use FINDOLOGIC\FinSearch\Export\ProductService;
+use FINDOLOGIC\FinSearch\Tests\Traits\DataHelpers\ProductHelper;
+use FINDOLOGIC\FinSearch\Tests\Traits\DataHelpers\SalesChannelHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Content\Product\ProductEntity;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Test\TestCaseBase\IntegrationTestBehaviour;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class ProductServiceTest extends TestCase
+{
+    use IntegrationTestBehaviour;
+    use SalesChannelHelper;
+    use ProductHelper;
+
+    /** @var SalesChannelContext|MockObject */
+    private $salesChannelContextMock;
+
+    protected function setUp(): void
```

Comment:
Add `$this->defaultProductService = $this->getDefaultProductService();` and use it in those tests.

Reply:
Still open?